engine/src/main/resources/org/hibernate/validator/ValidationMessages.properties

```diff
@@ -28,7 +27,8 @@ org.hibernate.validator.constraints.Range.message                   = must be be
 org.hibernate.validator.constraints.SafeHtml.message                = may have unsafe html content
 org.hibernate.validator.constraints.ScriptAssert.message            = script expression "{script}" didn't evaluate to true
 org.hibernate.validator.constraints.URL.message                     = must be a valid URL
+org.hibernate.validator.constraints.NumberPattern.message           = must match "{regexp}"
```

Comment:
See javax.validation.constraints.DecimalMax.message for a conditional usage. I'm not sure either about this. Let it alone for now, we can improve that later if users complain.